rendering/markup_content_test.go

```diff
@@ -0,0 +1,33 @@
+package rendering
+
+import "testing"
+import "github.com/stretchr/testify/assert"
+import "github.com/stretchr/testify/require"
+
+func TestGetDefaultMarkupFromNil(t *testing.T) {
```

Comment:
I think the functions `TestGetDefaultMarkupFromNil`, `TestGetMarkupFromValue`, and `TestGetMarkupFromEmptyValue` can be merged into one test function, don't you think?